docs/source/markdown/podman-image-prune.1.md

```diff
@@ -30,7 +30,7 @@ Do not provide an interactive prompt for container removal.
 
 Print usage statement
 
-## Examples ##
+## EXAMPLES ##
```

Comment:
Because consistency is everything:
```suggestion
## EXAMPLES
```

Reply:
Good catch, but to be fair, the inconsistent capitalization does show up in the rendered documentation. Extra `##` at the end of a heading don't make a difference in the output. But you're right, we may as well get it while we're here.